src/comp/comp.py

```diff
@@ -21,51 +21,53 @@ def __repr__(self):
     Human("David", 31),
 ]
 
+print(humans[0].name)
+
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with 'D':
 print("Starts with D:")
-a = []
+a = [human.name for human in humans if human.name[0] == 'D']
 print(a)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name ends in "e".
 print("Ends with e:")
-b = []
+b = [human.name for human in humans if human.name.endswith("e") == True]
 print(b)
 
 # Write a list comprehension that creates a list of names of everyone
 # whose name starts with any letter between 'C' and 'G' inclusive.
 print("Starts between C and G, inclusive:")
-c = []
+c = [human.name for human in humans if human.name[0] == 'C' or human.name[0] == 'D' or human.name[0] == 'E' or human.name[0] == 'F' or human.name[0] == 'G']
```

Comment:
This works but you could also do something like
```suggestion
c = [human.name for human in humans if human.name[0] in ['C', 'D', 'E' , 'F', 'G']
```